small cmake fixes

- remove hardcoded build type
- remove hardcoded '-o aslr', causes build failure:
cc1: error: too many filenames given; type 'cc1 --help' for usage

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 5d8ca13..e79551b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -7,7 +7,6 @@ endif()
 
 #{{{ Basic App setup
 set(PROJECT_NAME tagutil)
-set(CMAKE_BUILD_TYPE DEBUG)
 add_definitions(-DT_TAGUTIL_VERSION="3.1")
 project(${PROJECT_NAME} C)
 
@@ -31,7 +30,7 @@ include_directories(
 
 # CFLAGS
 add_compile_options(-std=c11 -Wall -Wextra)
-add_compile_options(-fstack-protector-strong -o aslr -fpie)
+add_compile_options(-fstack-protector-strong -fpie)
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pie")
 # Per build type flags.
 set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS} -O0 -g")
